Octokit.Reactive/Clients/ObservableCommitStatusClient.cs

```diff
@@ -28,10 +28,34 @@ public ObservableCommitStatusClient(IGitHubClient client)
         /// <returns></returns>
         public IObservable<CommitStatus> GetAll(string owner, string name, string reference)
         {
+            Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
+            Ensure.ArgumentNotNullOrEmptyString(name, "name");
+            Ensure.ArgumentNotNullOrEmptyString(reference, "reference");
```

Comment:
Rather than duplicating all these rules, `GetAll(owner, name, reference)` should just call `GetAll(owner, name, reference, ApiOptions.None)`